code/datums/traits/negative.dm

```diff
@@ -370,3 +370,97 @@
 		if(prob(1))
 			new/obj/item/reagent_containers/food/snacks/spaghetti/pastatomato(get_turf(H)) //now that's what I call spaghetti code
 
+/datum/quirk/smoker
+	name = "Smoker"
+	desc = "Sometimes you just really want a smoke. Probably not great for your lungs."
+	value = -1
+	gain_text = "<span class='danger'>You really could go for a smoke right about now.</span>"
+	lose_text = "<span class='notice'>You feel like you should quit smoking.</span>"
+	medical_record_text = "Patient is a current smoker."
+	var/obj/item/storage/fancy/cigarettes/preferred_type
+	var/where_lighter
+	var/where_cigs
+	var/obj/item/storage/fancy/cigarettes/cigs
+
+/datum/quirk/smoker/on_spawn()
+	preferred_type = pick(/obj/item/storage/fancy/cigarettes,
+		/obj/item/storage/fancy/cigarettes/cigpack_midori,
+		/obj/item/storage/fancy/cigarettes/cigpack_uplift,
+		/obj/item/storage/fancy/cigarettes/cigpack_robust,
+		/obj/item/storage/fancy/cigarettes/cigpack_robustgold,
+		/obj/item/storage/fancy/cigarettes/cigpack_carp,
+		/obj/item/storage/fancy/cigarettes/cigpack_syndicate)
```

Comment:
:eyes:

Reply:
oh well i guess you ll have to find them once you run out so i guess its balanced, maybe make the wrong band mood effect -7 so you really need to use that brand (-2 is barely noticeable)

Reply:
you can get them from hacked cigarette vending machines unless I'm remembering wrong.

Reply:
you can get them from space ruins, syndicate uplink, smuggler sachels and cargo contraband crates